packages/web-react/src/types/dropdown.ts

```diff
@@ -41,3 +41,9 @@ export interface SpiritDropdownProps extends DropdownProps, ChildrenProps {
 }
 
 export interface UncontrolledDropdownProps extends ChildrenProps, Omit<SpiritDropdownProps, 'isOpen' | 'onToggle'> {}
+
+export type DropdownTriggerComponentProps<E extends ElementType> = {
```

Comment:
```suggestion
export type DropdownTriggerProps<E extends ElementType> = {
```

Please rename this type to be consistent with other component types. I see the conflict with the [`DropdownTriggerProps`](https://github.com/lmc-eu/spirit-design-system/pull/1449/files#diff-5f651f6b8970dfbc54094ee39ac57b8f62ab87dfebdad67a3d7f069fd30dc1fcR13) so because it is used only in this file I suggest to rename it to the `DropdownTriggerRenderProps`. 

I also think that this will conflict with the incoming major release, so we must take care of this when rebasing the integration branch.